python/practices/practice-3/src/path1/letter_validation.py

```diff
@@ -0,0 +1,40 @@
+"""
+Enter a word and a string of letters, test the word using only the
+letters in the string
+"""
+
+
+import os
+
+
+def used_only_letters(word, string_letter):
```

Comment:
I think should use `allowed_letters` instead of `string_letter`

Reply:
Update the function name, ex: `uses_only`

Reply:
Done